packages/core/lib/Pipeline.ts

```diff
@@ -115,7 +147,7 @@ class Pipeline extends StreamObject {
 
   // stream interface
 
-  async _read(size) {
+  async _read(size: number) {
     try {
       await this.init()
```

Comment:
not related to this PR, but my IDE suggests that `await` is not needed

Reply:
Actually, it was related as you explained. Thanks!